### PR TITLE
kde-misc/akonadi-ews: New package

### PR DESCRIPTION
**Pull request by KrissN**
About a week ago I have released the initial version of a new Akonadi resource for Microsoft Exchange. I'd like to propose an ebuild for it into the KDE overlay.

### CODE REVIEW

**Comment by a17r** [comment]
CI is failing because PIM dependencies of akonadi-ews are masked - this would be fixed by adding it to profiles/package.mask/misc

File: kde-misc/akonadi-ews/akonadi-ews-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+HOMEPAGE="https://github.com/KrissN/akonadi-ews"
+
+if [[ ${KDE_BUILD_TYPE} = live ]] ; then
+	EGIT_REPO_URI="https://github.com/KrissN/${PN}.git"
+else
+	SRC_URI="https://github.com/KrissN/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="LGPL-2+"
+KEYWORDS="~amd64 ~x86"
```
Comment (a17r) [inline kde-misc/akonadi-ews/akonadi-ews-9999.ebuild]: needs to be empty for 9999 ;)

**Comment by johu** [comment]
Please rebase your commits according to topic and follow commit message format.

**Comment by johu** [comment]
Thanks. Picked and splitted package mask into separate commit.

**Pull request closed without merging**